src/index.html

```diff
@@ -7,8 +7,88 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header main">
+      <a href="#main" class="logo">
+        <img src="images/logo.png" alt=" ">
```

Comment:
Alt attribute shouldn't be empty
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/bad469cd-79b0-4826-ab8b-89ad2629c5e4)

Reply:
it is clear